### ISSUE TITLE
Turn the `m` on the logo (`favicon.ico`) into a `c`

### ISSUE THREAD

**Issue by lenormf** [mosra/corrade#110]
Hi,

This library has the same logo as Magnum. It'd be nice to make it differ (the simplest change would be to turn the `m` into a `c`), for external use.

Thanks!

**Comment by mosra** [comment]
Yeah, the projects need actual logos :) This is just a placeholder that's just a differently colored `m` from https://mcss.mosra.cz/, which is accidentally also very similar to what MSDN used to have.

However I'm afraid I don't have time to work on "branding" anytime soon as there are other tasks with much higher priority. Possibly mid-2021, but don't take my word for that.

**Comment by lenormf** [comment]
No problem. I have notifications on, feel free to close this issue once you've gotten around to doing it.
